pkg/routeagent/controllers/route/route_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package routecontroller_test
```

Comment:
I wonder if we can setup a a single test suite for the whole project, or if we need to do package by package.

Reply:
This needs to explored.

Reply:
I think it needs to be per package b/c it relies on package initializers to register the specs, ie all specs must be registered either via global vars or init() functions before RunSpecs is called.